pkg/controller/nodenetworkconfigurationpolicy/conditions/manager.go

```diff
@@ -48,20 +57,56 @@ func (m *Manager) NotifySuccess() {
 	}
 }
 
+func (m *Manager) initializeEnactment() (*nmstatev1alpha1.NodeNetworkConfigurationEnactment, error) {
+
+	node := corev1.Node{}
+	err := m.client.Get(context.TODO(), types.NamespacedName{Name: m.nodeName}, &node)
+	if err != nil {
+		return nil, errors.Wrap(err, fmt.Sprintf("cannot find node %s", m.nodeName))
```

Comment:
It is not that we cannot find it, but we cannot get it. "Find" suggest that we looked for it but it does not exist, which should not be the case (we run on the node, so it probably exists). Could you change the message?